tests/core/elastic_sync_test.py

```diff
@@ -0,0 +1,189 @@
+import simplejson as json
+
+from superdesk.tests.asyncio import AsyncTestCase
+from superdesk.core.resources.fields import HTML, Keyword, ObjectId
+from superdesk.core.elastic.common import SearchRequest
+
+from .modules.users import User, RelatedItems, Category
+
+
+test_user_1 = User(
+    id="user_1",
+    first_name="John",
+    last_name="Doe",
+    bio=HTML("<p>This is my bio</p>"),
+    code=Keyword("my codes"),
+    categories=[
+        Category(qcode="sports", name="Sports", scheme="category"),
+        Category(qcode="swimming", name="Swimming", scheme="sports"),
+    ],
+    related_items=[
+        RelatedItems(id=ObjectId(), link_type=Keyword("text"), slugline=HTML("sports-results")),
+        RelatedItems(id=ObjectId(), link_type=Keyword("photo"), slugline=HTML("sports-results")),
+    ],
+)
```

Comment:
We should put this into a separate function instead of having it statically setup, specially because it is used in multiple tests.

Reply:
I will add a fixtures folder to `tests/core` that we can store model instances for use throughout test cases (we do something similar in Newshub, so test cases work against a common group of fixtures)